src/import-data/ImportDataDestination.ts

```diff
@@ -219,7 +220,9 @@ export const ImportDataDestination = (props: ImportDataDestinationProps): ReactN
         h(WorkspacePolicies, {
           workspace: selectedWorkspace,
           noCheckboxes: true,
-          endingNotice: isProtectedData ? div(['Importing this data may add additional access controls']) : undefined,
+          endingNotice: importRequiresSecurityMonitoring
+            ? div(['Importing this data may add additional access controls'])
+            : undefined,
```

Comment:
An example of where the current "protected data" term is creating misleading messaging.